lib/demo_web/live/user_live/index.ex

```diff
@@ -11,7 +11,7 @@ defmodule DemoWeb.UserLive.Index do
     {:ok, assign(socket, page: 1, per_page: 5)}
   end
 
-  def handle_params(params, url, socket) do
+  def handle_params(params, _url, socket) do
```

Comment:
add this line
```
if connected?(socket), do: Demo.Accounts.subscribe()
```

Reply:
Thank you thank you!  Updated and nice job 👍